madara-prover-common/src/models.rs

```diff
@@ -141,6 +142,31 @@ pub struct Proof {
     pub proof_parameters: ProverParameters,
     pub prover_config: ProverConfig,
     pub public_input: PublicInput,
+    #[serde(skip)]
+    pub working_dir: Option<ProverWorkingDirectory>,
```

Comment:
@odesenfans This seems a bit hacky, but it should be efficient. I moved `ProverWorkingDirectory` here and added this to the main Proof struct so that the temp files would stick around for the duration of the proof itself, which works well since they are needed after calling `cpu_air_prover`.